zkevm-circuits/Cargo.toml

```diff
@@ -17,7 +17,7 @@ eth-types = { path = "../eth-types" }
 gadgets = { path = "../gadgets" }
 ethers-core = "0.17.0"
 ethers-signers = "0.17.0"
-mock = { path = "../mock" }
+mock = { path = "../mock", optional = true }
```

Comment:
anybody know `optional` functionality here or share some document for it? thanks .

Reply:
It is explained here, in the second example of the section: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies